wizard/views/load.py

```diff
@@ -0,0 +1,22 @@
+import streamlit as st
+import mlcroissant as mlc
+import tempfile
+from pathlib import Path
+import os
+
+def render_load():
+    col1, col2 = st.columns([1,2], gap="small")
```

Comment:
The Python code is not formatted (spacing, EOF, import orders). I will set up black/isort in the CI/CD.

Can you also install the `black` and `isort` extensions to have your code auto-formatted?